index.md

```diff
@@ -107,7 +107,7 @@ description: Vanilla OS is an Immutable Linux-based distribution which aims to p
             </a>
         </div>
         <div class="hero-image" data-scroll>
-            <img class="animate-on-scroll" data-animation="fadeInLeft" src="/assets/images/home-desk-apx.svg" alt="Vanilla OS - Immutable!" />
+            <img class="animate-on-scroll" data-animation="fadeInLeft" src="/assets/images/home-desk-apx.svg" alt="Terminal written  “$ apx install nice-stuff” and “$ abroot exec apt install nice-stuff” in different rows." />
```

Comment:
```suggestion
            <img class="animate-on-scroll" data-animation="fadeInLeft" src="/assets/images/home-desk-apx.svg" alt="A simple terminal application showing the commands "apx install nice-stuff" and "abroot exec apt install nice-stuff"" />
```

I think this image has some value outside of being decorative, since it showcases a bit of how `apx` works.